apps/backoffice/src/lib/be/institutions/business.ts

```diff
@@ -1,16 +1,26 @@
+import { GroupPagination } from "@/generated/api/GroupPagination";
 import { Institution as BackofficeInstitution } from "@/generated/api/Institution";
 import { UserAuthorizedInstitution } from "@/generated/api/UserAuthorizedInstitution";
 import { UserAuthorizedInstitutions } from "@/generated/api/UserAuthorizedInstitutions";
 import { Institution as SelfcareInstitution } from "@/generated/selfcare/Institution";
 
-import { getInstitutionById, getUserAuthorizedInstitutions } from "./selfcare";
+import { ManagedInternalError } from "../errors";
+import {
+  getInstitutionById,
+  getInstitutionGroups,
+  getUserAuthorizedInstitutions,
+} from "./selfcare";
 
 // Type utility to extract the resolved type of a Promise
 type PromiseValue<T> = T extends Promise<infer U> ? U : never; // TODO: move to an Utils monorepo package
 
 // Type utility to extract the resolved type of a ReadonlyArray of Promises
 type ReadonlyArrayElementType<T> = T extends readonly (infer U)[] ? U : never; // TODO: move to an Utils monorepo package
 
+// Type utility to extract the resolved type of an array of Promises
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+type ArrayElementType<T> = T extends (infer U)[] ? U : never; // TODO: move to an Utils monorepo package
+
```

Comment:
question: what are these lines for?

Reply:
lint gives an error because is not used , but since this type was already there before although it was not used i didn't remove it and just added the ignore rule

Reply:
sorry i was wrong it was not there before this PR i think i was doing some tests and forgot about it later , i'll remove it

Reply:
[4ce0326](https://github.com/pagopa/io-services-cms/pull/1007/commits/4ce0326ee02d74d1979064291f7fad04d82ab937)